raster/parquet-raster.md

```diff
@@ -0,0 +1,122 @@
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED",  "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt).
+
+## Raster columns
+
+A raster column MUST be stored as a `struct` type column in parquet files. The `struct` type MUST contain the fields defined in the following table. The `raster` column MUST be stored in the root level of the parquet file.
+
+Each raster column must also have a corresponding `Geometry` or `Geography` type column, stored in the top level of the parquet file. The name of the geometry column MUST be specified in the `geometry` field of the raster column metadata.
+
+## Raster Representation
+
+The raster data model is largely inspired by the WKB raster encoding of PostGIS but extracts the raster metadata out of the binary encoding.
+
+### Raster value
+
+A raster value is composed by the following components:
+
+| Field        | Parquet Physical Type | Parquet Logical Type | Description                                                             |
+|--------------|-----------------------|----------------------|-------------------------------------------------------------------------|
+| `endianness` | `boolean`             |                      | **REQUIRED.** True: little endian; False: big endian                    |
+| `crs`        | `BYTE_ARRAY`          | UTF8                 | **OPTIONAL.** The coordinate reference system of the raster             |
+| `scale_x`    | `DOUBLE`              |                      | **REQUIRED.** The scale factor of the raster in X direction             |
+| `scale_y`    | `DOUBLE`              |                      | **REQUIRED.** The scale factor of the raster in Y direction             |
+| `ip_x`       | `DOUBLE`              |                      | **REQUIRED.** The X coordinate of the upper left corner of the raster   |
+| `ip_y`       | `DOUBLE`              |                      | **REQUIRED.** The Y coordinate of the upper left corner of the raster   |
+| `skew_x`     | `DOUBLE`              |                      | **REQUIRED.** The skew factor of the raster in X direction              |
+| `skew_y`     | `DOUBLE`              |                      | **REQUIRED.** The skew factor of the raster in Y direction              |
+| `width`      | `INT32`               |                      | **REQUIRED.** The width of the raster in pixels                         |
+| `height`     | `INT32`               |                      | **REQUIRED.** The height of the raster in pixels                        |
+| `bands`      | `BYTE_ARRAY`          | List<BYTE_ARRAY>     | **REQUIRED.** The bands of the raster. See the band data encoding below |
```

Comment:
What is the best type of an array of `Byte_array` here? @Kontinuation

Reply:
We don't have to store `bands` as Parquet logical list type. `bands` field can be defined as `repeated` to store multiple BYTE_ARRAYs sequentially.

Reply:
OK. Let's leave it as is for now?